application-services/custom/json-logic/res/configuration.yaml

```diff
@@ -0,0 +1,36 @@
+Writable:
+  LogLevel: 'INFO'
+  Writable.Telemetry:
+    Interval: "30s"
+    PublishTopicPrefix : "edgex/telemetry" # /<service-name>/<metric-name> will be added to this Publish Topic prefix
+
+
+Service:
+  Host: 'localhost'
+  Port: 59780 # Adjust if running multiple examples at the same time to avoid duplicate port conflicts
+  StartupMsg: 'This is a sample application service using the JSON logic transform'
+
```

Comment:
Missing MessageBus section for client ID override

```
MessageBus:
  Disabled: false  # Set to true if not using edgex-messagebus Trigger below and Metrics are not needed
  Optional:
    ClientId: "app-json-logic"
```

Reply:
done